torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -2373,15 +2373,39 @@ Error PyTorchModelLoader::loadLog(const torch::jit::Node *ptNode) {
 Error PyTorchModelLoader::loadMax(const torch::jit::Node *ptNode) {
   auto inputs = ptNode->inputs();
   auto outputs = ptNode->outputs();
-  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 2, outputs, 1));
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, -2, outputs, -1));
 
-  glow::NodeValue lhs;
-  ASSIGN_VALUE_OR_RETURN_ERR(lhs, getGlowNodeValueForValue(inputs[0]));
-  glow::NodeValue rhs;
-  ASSIGN_VALUE_OR_RETURN_ERR(rhs, getGlowNodeValueForValue(inputs[1]));
+  glow::NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(input, getGlowNodeValueForValue(inputs[0]));
```

Comment:
As we are now having more than 2 inputs some time, it is quite important to explicitly name each input. Please reference other ops for example.

Reply:
Also, in this case we have two different formats for one op, we can create two enum struct, like MaxInputsBasic and MaxInputAxis (Choose the name you like :) )

Reply:
Done.